src/parser_internal.h

```diff
@@ -348,6 +377,32 @@ LY_ERR lyd_parse_lyb(const struct ly_ctx *ctx, const struct lysc_ext_instance *e
         struct lyd_node **first_p, struct ly_in *in, uint32_t parse_opts, uint32_t val_opts, uint32_t int_opts,
         struct ly_set *parsed, ly_bool *subtree_sibling, struct lyd_ctx **lydctx_p);
 
+#ifdef ENABLE_CBOR_SUPPORT
```

Comment:
Again not necessary, the function can exist and fail. I would like to keep these macros to a minimum.